backend/otel/otel.go

```diff
@@ -380,7 +395,7 @@ func (o *Handler) submitProjectLogs(ctx context.Context, projectLogs map[string]
 	for projectID, logRows := range projectLogs {
 		var hasBackendLogs bool
 		for _, logRow := range logRows {
-			if logRow.IsBackend() {
+			if logRow.Source == modelInputs.LogSourceBackend {
```

Comment:
fine to do the check inline, but why not keep a helper method for the backend check?

Reply:
Didn't really feel like it was needed since there's only two possible values for `Source` i.e. it feels like an unnecessary abstraction.